### ISSUE TITLE
[Feature Request] Per project config

### ISSUE THREAD

**Issue by mslalith** [zellij-org/zellij#2308]
Provide an option to load zellij based on the config file defined in the root of the project.
For example, say I want to run certain set of commands (cargo run, etc) on opening a Rust projects whereas for react they can be a different set of commands

**Comment by Zykino** [comment]
This is what I was about to propose, commit one or more zellij layout in your repo. This way you can even have a `zellij/dev.kdl` and another `zellij/publish.kdl`, … Then load the one you want to use at the moment.

Pssst, it’s `kdl` with a d, like cuddle ;)

**Comment by mslalith** [comment]
@mark2185 yes that should be enough. Honestly that would be simple a solution to this. One thing we can improve is as what @Zykino mentioned, we can have multiple kdl files and having zellij detect these and show a selection with these options. Upon user selection we could load that config would be a great addition

**Comment by Zykino** [comment]
` zellij action new-tab -l <personal layout>` already works. Where "personal layout" is in `~/.config/zellij/layouts/<personal layout>.kdl`.

I don’t think zellij should look in a predefined place in your repo:
Does your app have a layout folder for the app itself?
Are you in the root of the repo when starting a command? In a submodule?
…

Maybe looking for a `zellij-layout` folder or a `zellij.config` file and reading from it? But still the sames questions are up in the air.
And letting peoples adapt `zellij action new-tab -l ./zellij-l/dev.kdl` with the auto-completion of their shell is not that far from what you want (without overriding personal layout with the repos's ones).
